kalite/control_panel/views.py

```diff
@@ -558,6 +560,27 @@ def _get_user_usage_data(users, groups=None, period_start=None, period_end=None,
     return (user_data, group_data)
 
 
+def check_meta_data(facility_id):
+    '''Checks whether any metadata is missing for the specified facility.
+
+    Args: 
+      facility_id (str): id of facility to check
+ 
+    Returns:
+      bool: True if one or more metadata fields are missing'''
+
+    facility_data = {}
+    facility_entry = Facility.objects.filter(id=facility_id)[0]
```

Comment:
If this is what we wanted to do, you should use 'get' not 'filter'. However, you should be passing in the facility itself to avoid redundant database lookups.